### DISCUSSION TITLE
Group Size

### DISCUSSION THREAD

**Discussion by thespica93** [forkineye/ESPixelStick#570] [General]
Hello,

I have more of a question than feature request. 

I have a prop that has 65 pixels and a second prop that is 100 pixels. I would like to run both props from the 1 pixelstick.  I am trying to lower my channel count. Can I do 2 separate group sizes, one for the first prop and then another for the second prop.

Thank you
-Nick

**Comment by MartinMueller2003** [comment]
It depends. The ESPixelStick supports a group size setting per port. The answer for a single port solution would be - no. If you get an ESP32 based PixelStick based on a multi port platform like the Dig-Quad from quinled. then you could put the two props on different ports and use different group sizes.
